assets/src/design-system/components/dropDown/select/components.js

```diff
@@ -86,6 +83,7 @@ export const SelectButton = styled.button(
         color: ${theme.colors.fg.disable};
       }
     }
+    ${selectButtonStylesOverride};
```

Comment:
Added ability to override select button styles since this was part of the original reason the `DropDown` wasn't used for the animation effects.